lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -238,9 +238,12 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     inDialog: boolean = false;
     _chosenNode: Node [] = null;
     folderIdToShow: string | null = null;
-    breadcrumbFolderTitle: string | null = null;
     startSiteGuid: string | null = null;
 
+    @Output()
+    breadcrumbFolderTitleEvent = new EventEmitter<string | null>();
```

Comment:
Do we need the `| null`? Also can you leave an empty line between the event emitter and the next variable declaration? It would be nice to move this on top of the class under all the other @Outputs , usually all the Inputs and Outputs are grouped together in our components which I think has it's pros

Reply:
What happens if breadcrumbFolderTitle is null? The original property has type `string | null`, implying there would be a chance we'd emit null value

Reply:
theoretically every Event emitter could emit null if something goes wrong. To be honest not sure if it's needed as a type declaration. @mauriziovitale any idea?